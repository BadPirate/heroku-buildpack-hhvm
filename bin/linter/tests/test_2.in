Bad Value {
  BadValue
}
# Extra brace, why not?
} 

Foo {
  * {
    this : is : invalid
    value = 
  }
# Missing a brace too, like a boss
