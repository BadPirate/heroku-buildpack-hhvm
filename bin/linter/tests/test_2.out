-----> Unexpected node name in test_2.in on line 1
       Bad Value {
-----> Bad config value in test_2.in on line 2
       BadValue
-----> Unexpected closing brace found on line 5
-----> Bad config value in test_2.in on line 9
       this : is : invalid
-----> Bad config value in test_2.in on line 10
       value =
-----> Braces don't match. Found 1 more than expected.
